rpm/centos-8/Dockerfile

```diff
@@ -20,7 +20,8 @@ ENV SUITE=${SUITE}
 
 # In aarch64 (arm64) images, the altarch repo is specified as repository, but
 # failing, so replace the URL.
-RUN if [ -f /etc/yum.repos.d/CentOS-Sources.repo ]; then sed -i 's/altarch/centos/g' /etc/yum.repos.d/CentOS-Sources.repo; fi
+RUN if [ -f /etc/yum.repos.d/CentOS-Linux-Sources.repo ]; then sed -i 's/altarch/centos/g' /etc/yum.repos.d/CentOS-Linux-Sources.repo; fi
+RUN if [ -f /etc/yum.repos.d/CentOS-Linux-PowerTools.repo ]; then sed -i 's/altarch/centos/g' /etc/yum.repos.d/CentOS-Linux-PowerTools.repo; fi
```

Comment:
Not sure if this one needed the s/altarch/centos/

The variant in the containerd-packaging used this to enable/disable the repo, which is what this dockerfile does below with `yum-config-manager --set-enabled PowerTools`

Reply:
So we could do https://github.com/docker/containerd-packaging/commit/6b278d494fa5a5c4342ceff09fbbb01a1e8f0cad#diff-27a5d8094b1603f8faf9155b428e8607c1b7607d589868bf14a59b45d7e4eb16R48

_instead off_ `yum-config-manager --set-enabled PowerTools`